Bottom: 1acbccbddc2cdd476e8cdcc0f9071f10b5cf4278
Top:    8bd53ef973ba69bb5e71de4d5884f7cad7a85bb6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-26 15:34:34 -0500

Refresh of add-docker-option

---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index e4b17643..a8251373 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -31,11 +31,16 @@
           ldap_user_id_attribute: 'uid'
           ldap_user_real_name_attribute: 'cn'
           ldap_user_email_attribute: 'mail'
+          ldap_map_groups_as_roles: yes
+          ldap_map_groups_as_roles_type: "static"
+          ldap_user_memberof_attribute: ""
           ldap_group_base_dn: 'ou=group'
           ldap_group_object_class: 'posixGroup'
-          ldap_group_id_attribute: 'gidNumber'
+          ldap_group_id_attribute: 'cn'
           ldap_group_member_attribute: 'memberUid'
           ldap_group_member_format: '${username}'
+          ldap_user_subtree: false
+          ldap_group_subtree: true
 
       nexus_privileges:
         - name: all-repos-read  # used as key to update a privilege
diff --git a/ansible-playbook/nexus.yml b/ansible-playbook/nexus.yml
new file mode 100644
index 00000000..5030618a
--- /dev/null
+++ b/ansible-playbook/nexus.yml
@@ -0,0 +1,13 @@
+---
+    
+- name: Install Sonatype Nexus Repository Manager 3
+  hosts: daedalus
+  become: true
+  become_user: root
+  become_method: sudo
+  tasks:
+  - set_fact:
+      openssl_dv_name: "openssl_DV"
+  - name: Run Nexus role
+    include_role:
+      name: nexus3
